cmake/compiler/compiler_flags_template.cmake

```diff
@@ -13,6 +13,8 @@ set_compiler_property(PROPERTY optimization_speed)
 
 set_compiler_property(PROPERTY optimization_size)
 
+set_compiler_property(PROPERTY optimization_min_size)
```

Comment:
nit: same comment regarding maybe using the same `optimization_size` prefix here (and therefore calling this `optimization_size_aggressive`)?

Reply:
Done!